.github/workflows/publish_pypi.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish package to PyPi
+
+on:
+   release:
+     types: [published]
+
+jobs:
+
+   push:
+     runs-on: ubuntu-20.04
+
+     steps:
+       - name: Checkout
+         uses: actions/checkout@v2
+       - name: setup python
+         uses: actions/setup-python@v2
+         with:
+           python-version: 3.5
+
+       - name: Install pip
+         run: pip install -r requirements/pip.txt
+
+       - name: Install Dependencies
+         run: pip install setuptools wheel
```

Comment:
this is unnecessary, these dependencies should already be installed by `pip.txt`